src/ScintillaWnd.cpp

```diff
@@ -829,43 +826,32 @@ void CScintillaWnd::UpdateLineNumberWidth()
 
 void CScintillaWnd::SaveCurrentPos(CPosData& pos)
 {
```

Comment:
again, the comment clearly states to NOT save the positions. Why did you remove that?

Reply:
Because I'm not sure the comment is correct any longer. See below.